lvgl-codegen/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "lvgl-codegen"
-version = "0.5.2"
+version = "0.6.0"
 description = "Code generation based on LVGL source code"
 authors = ["Rafael Caricio <crates.lvgl@caric.io>"]
```

Comment:
You can add your name in those.

Reply:
:o ty!! I will for the next release then, thanks so much